snippet-service/service/service.go

```diff
@@ -77,6 +77,9 @@ func newHttpService(ctx context.Context, config HTTPConfig, validatorConfig vali
 }
 
 func newGrpcService(ctx context.Context, config grpcService.Config, ss *grpcService.SnippetServiceServer, fileWatcher *fsnotify.Watcher, logger log.Logger, tracerProvider trace.TracerProvider) (*grpcService.Service, func(), error) {
+	if err := config.Authorization.Validate(); err != nil {
```

Comment:
This seems a bit weird, no other config has the `Validate` method called inside the service.go why is it needed here?

Reply:
fixed